fairytale.md

```diff
@@ -5,6 +5,7 @@
 In a faraway future, in a distant land  
 lives the *Squonk*
 
+The average Squonk is 7 meters tall and screams loud.
```

Comment:
I think this line should be removed. I removed it in the version you merged, so I'd prefer not to have it. If you can't do this change yourself by the time I perform the merge I will do it myself. It will not affect the grade.